### PR TITLE
feat: document store with wallet setup

### PR DESCRIPTION
**Pull request by yehjxraymond**

### CODE REVIEW

File: docs/verifiable-document/document-store.md [hunk 1]
```diff
@@ -4,8 +4,50 @@ title: Deploying Document Store Smart Contract
...
 
-Overview
+The document store is a smart contract on the Ethereum network that records the issuance and revocation status of OA documents. In this guide, we will deploy a document store smart contract on the Ethereum `ropsten` network which is a test network that does not require actual ethers for transactions.
 
-## Header
+## Pre-requisite
 
-TBD
+- Chrome browser
+- Metamask connected to `ropsten` network
```
Comment (Nebulis) [inline docs/verifiable-document/document-store.md]: just Metamask ? there is a step to switch to ropsten

Do you want to add a link t the appendix ?

Comment (Nebulis) [reply docs/verifiable-document/document-store.md]: you can ignore the second comment there is a link above =)

File: docs/verifiable-document/document-store.md [hunk 2]
```diff
@@ -4,8 +4,50 @@ title: Deploying Document Store Smart Contract
...
+
+![Confirming Transaction](/docs/verifiable-document/document-store/confirmation.png)
+
+After clicking on "Deploy", you will be asked to confirm the transaction in a separate popup. Click "Confirm" to continue.
+
+![Successful deployment](/docs/verifiable-document/document-store/success.png)
+
+Once your document store smart contract has been successfully deployed, you will see the success message with the `Store Address`.
+
+Save this value for future reference.
```
Comment (Nebulis) [inline docs/verifiable-document/document-store.md]: Should we emphasize on this ? To show it's important (it's a very important information and is hard to retrieve for new comers)

File: docs/verifiable-document/document-store.md (hunk 1 above)
Comment (yehjxraymond) [reply docs/verifiable-document/document-store.md]: Sorry, I dont get the first comment either...

File: docs/verifiable-document/document-store.md (hunk 2 above)
Comment (yehjxraymond) [reply docs/verifiable-document/document-store.md]: Moved it in a quote

**Pull request merged**